### PR TITLE
[OTE-738] update name generation with manual list

### PR DESCRIPTION
**Pull request by affanv14**
### Changelist
[Describe or list the changes made in this PR]

### Test Plan
[Describe how this PR was tested (if applicable)]

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new method for generating usernames using a combination of random adjectives and nouns, enhancing uniqueness and descriptiveness.
	- Added configuration options for maximum username length.
	- Introduced comprehensive lists of adjectives and nouns for more expressive outputs.

- **Chores**
	- Removed unused dependency to streamline the project and reduce potential vulnerabilities.

### CODE REVIEW

File: indexer/services/roundtable/src/tasks/subaccount-username-generator.ts [hunk 1]
```diff
@@ -32,3 +35,16 @@ export default async function runTask(): Promise<void> {
...
+  const randomAdjective: string = adjectives[randomInt(0, adjectives.length)];
+  const randomNoun: string = nouns[randomInt(0, nouns.length)];
+  const randomNumber: string = randomInt(0, 1000).toString().padStart(
+    config.SUBACCOUNT_USERNAME_NUM_RANDOM_DIGITS, '0');
+
+  const capitalizedAdjective: string = randomAdjective.charAt(
+    0).toUpperCase() + randomAdjective.slice(1);
+  const capitalizedNoun: string = randomNoun.charAt(0).toUpperCase() + randomNoun.slice(1);
+
+  return `${capitalizedAdjective}${capitalizedNoun}${randomNumber}`;
```
Comment (jerryfan01234) [inline indexer/services/roundtable/src/tasks/subaccount-username-generator.ts:49]: Can you include a unit test that checks that the capitalization, number and length conventions are followed?

File: indexer/services/roundtable/src/tasks/subaccount-username-generator.ts [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+import adjectives from '../lib/adjectives.json';
+import nouns from '../lib/nouns.json';
 
 export default async function runTask(): Promise<void> {
   const subaccounts:
   SubaccountsWithoutUsernamesResult[] = await
   SubaccountUsernamesTable.getSubaccountsWithoutUsernames();
   for (const subaccount of subaccounts) {
-    const username: string = generateUsername('', config.SUBACCOUNT_USERNAME_NUM_RANDOM_DIGITS);
+    const username: string = generateUsername();
```
Comment (jerryfan01234) [inline indexer/services/roundtable/src/tasks/subaccount-username-generator.ts:18]: are duplicate usernames allowed?

File: indexer/services/roundtable/src/tasks/subaccount-username-generator.ts [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
     try {
       // if insert fails, try it in the next roundtable cycle
       // There are roughly 50 Billion possible usernames with 3 random digits
```
Comment (jerryfan01234) [inline indexer/services/roundtable/src/tasks/subaccount-username-generator.ts:21]: is 50 billion for the old package?

Comment (Christopher-Li) [reply indexer/services/roundtable/src/tasks/subaccount-username-generator.ts:21]: ~250*350*1000 = 87.5M now

**Pull request merged**